MAUI/TextInputLayout/Getting-Started.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: post
+title: Getting Started with .NET MAUI Text Input Layout control | Syncfusion
+description: Learn here about getting started with Syncfusion .NET MAUI Text Input Layout (SfTextInputLayout) control, its elements and more.
+platform: maui
+control: SfTextInputLayout
+documentation: ug
+---
+
+# Getting Started with .NET MAUI Text Input Layout (SfTextInputLayout)
+
+This section explains the steps required to configure the `.NET MAUI Text Input Layout` (SfTextInputLayout) control with floating label.
+
+## Creating an application using the .NET MAUI Text Input Layout
+
+This section explains the steps required to work with the `SfTextInputLayout` control for .NET MAUI.
+
+## Adding .NET MAUI Text Input Layout reference
+
+The Syncfusion .NET MAUI controls are available in [Nuget.org](https://www.nuget.org/). To add `.NET MAUI Text Input Layout` to your project, open the NuGet package manager in Visual Studio, search for [Syncfusion.Maui.Core](https://www.nuget.org/packages/Syncfusion.Maui.Core/) and then install it.
```

Comment:
The Syncfusion .NET MAUI controls are available in the [Nuget.org](https://www.nuget.org/). To add `.NET MAUI Text Input Layout` to your project, open the NuGet package manager in Visual Studio, search for [Syncfusion.Maui.Core](https://www.nuget.org/packages/Syncfusion.Maui.Core/) and then install it.

Reply:
Addressed the changes